test/templateConfigValidator.test.js

```diff
@@ -1,7 +1,7 @@
 const { validateTemplateConfig } = require('../lib/templateConfigValidator');
 const fs = require('fs');
 const path = require('path');
-const streetlightYAML = fs.readFileSync(path.resolve(__dirname, './docs/streetlights.yml'), 'utf8');
+const streetlightYAML = fs.readFileSync(path.resolve(__dirname, './docs/dummy.yml'), 'utf8');
```

Comment:
Maybe rename to `dummyYAML` since streetlight is no longer a thing in the generator :thinking:?

Reply:
of course, you are right, great catch, on it